main.go

```diff
@@ -25,6 +25,10 @@ type Book struct {
 	Price     *float32  `json:"price"`
 	Inventory *int      `json:"inventory"`
 }
+type errorResponse struct {
+	Error_code    int    `json:"error_code"`
+	Error_message string `json:"error_message"`
+}
```

Comment:
The JSON tag will handle the name of the fields in the JSON document.
Go doesn't use snake case `snake_case` for the names, we use `CamelCase`.
```suggestion

type errorResponse struct {
	Code    int    `json:"error_code"`
	Message string `json:"error_message"`
}

Reply:
Done.